tests/api/test_endpoints.py

```diff
@@ -180,6 +180,77 @@ def test_post_clubs__failure(server):
     assert 'error' in response.json
 
 
+def test_put_club_image__success(server):
+    # POST a dummy user to add a profile image to
+    token = util.create_jwt(2, server.config.secret)
+    data = FormData()
+    data.add_field('image', open('tests/testdata/large-logo.png', 'rb'))
+    _, response = server.app.test_client.put(
+        '/clubs/test/images/profile',
+        data=data,
+        headers={
+            'Authorization': token,
+        })
+    assert response.status == 200
+
+
+def test_put_club_image__failure(server):
+    # No such club
+    token = util.create_jwt(2, server.config.secret)
+    data = FormData()
+    _, response = server.app.test_client.put(
+        '/clubs/newtest/images/profile',
+        data=data,
+        headers={
+            'Authorization': token,
+        })
+    assert response.status == 404
+    # Invalid image name
+    _, response = server.app.test_client.put(
+        '/clubs/test/images/$%^&*(',
+        data=data,
+        headers={
+            'Authorization': token,
+        })
+    assert response.status == 400
+
+
+def test_get_club_image__success(server):
+    _, response = server.app.test_client.get('/clubs/test/images/profile')
+    assert response.status == 200
+
+
+def test_get_club_image__failure(server):
+    # No such club
+    _, response = server.app.test_client.get('/clubs/newTest/images/profile')
+    assert response.status == 404
+
+
+def test_delete_club_image__success(server):
+    token = util.create_jwt(2, server.config.secret)
+    _, response = server.app.test_client.delete(
+        '/clubs/test/images/profile', headers={
+            'Authorization': token,
+        })
+    assert response.status == 200
+
+
+def test_delete_club_image__failure(server):
+    token = util.create_jwt(2, server.config.secret)
+    # No such image
+    _, response = server.app.test_client.delete(
+        '/clubs/test/images/profile', headers={
+            'Authorization': token,
+        })
+    assert response.status == 404
+    # # Forbidden (user is trying to delete image of an unrelated club)
+    # _, response = server.app.test_client.delete(
+    #     '/clubs/newtest/images/profile', headers={
+    #         'Authorization': token,
+    #     })
+    # assert response.status == 403
```

Comment:
Should this be commented out?

Reply:
@kevinyok Don't for get about this.

Reply:
@bfbachmann  I made the change as discussed.